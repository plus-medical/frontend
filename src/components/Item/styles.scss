@import '../styles/Global.scss';

.item{
  display:grid;
  box-sizing: border-box;
  grid-template-columns: 83px 1fr 15px;
  width: $size-card-width;
  justify-content: center;
  border-radius:0 20px 0 20px;
  background-color:$background-color-card;
  z-index: 1;
  margin-bottom: 10px;
  max-height: 83px;
  animation: 300ms ease-in entry_item;
}

.item__icon{
  position: relative;
  top: -25px;
  right: -59px;
  color: $background-color-card;
  font-size: $border-search;
}

.item__content{
  display: flex;
  justify-content: flex-start;
  flex-direction: column;
  flex-wrap: wrap;
  padding: 5px 10px;
  color: $font-color
}

.item__image{
  border-radius: 0px 15% 0px 15%;
  object-fit: contain;
  width: 83px;
}

.item__name{
  font-size:$font-size-nameCard;
  font-weight: bold;
  margin:0px 0px 5px 0px;
}

.item__dni{
  font-size:$font-size-dni;
  font-weight: 300;
  margin:0px 0px;
  color: $primary-color-1;
}

.item__profile{
  font-weight: 500;
  font-size:$font-size-profile;
  margin:5px 0px;
}

.item__indicator{
  height: $size-img-height;
  background-color: $color-great;
  border-radius: 0 15px 0 0;
}

@keyframes entry_item{
  from {
    opacity: 0;
  }  
  to{
    opacity: 1;
  }
}
