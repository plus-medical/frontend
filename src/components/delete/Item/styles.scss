.Item{
  display:grid;
  grid-template-columns: 30% 66% 4%;
  width: 20rem;
  align-items: center;
  justify-content: center;
  border-radius:0 20px 0 20px;
  background-color:white;
}

.Item--indicator{
  height: 5.2rem;
  background-color: green;
  border-radius: 0 15px 0 0;
}
