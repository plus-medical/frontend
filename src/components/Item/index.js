import React from 'react'
import { Link } from 'react-router-dom'
import './styles.scss'

export function Item () {
  return (
    <Link to='/user'>
      <li className='Item'>
        <img
          className='Item__Image'
          src='https://pbs.twimg.com/profile_images/1055299470148227072/xeEopDm__400x400.jpg'
          alt='photo'
        />
        <article className='Item__content'>
          <h5 className='Item__name'>name</h5>
          <label className='Item__dni'>1234322</label>
          <h6 className='Item__profile'>Profile</h6>
        </article>
        <section className='Item__indicator' />
      </li>
    </Link>
  )
}
