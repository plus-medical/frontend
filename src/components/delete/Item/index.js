import React from 'react'
import './styles.scss'

import { ImageSearch } from '../../atoms/images'
import { NameSearch, DescriptionSearch } from '../../atoms/titles'
import { DocumentSearch } from '../../atoms/texts'

export function Item () {
  return (
    <li className='Item'>
      <ImageSearch />
      <article className='Item--content'>
        <NameSearch />
        <DocumentSearch />
        <DescriptionSearch />
      </article>
      <section className='Item--indicator' />
    </li>
  )
}
