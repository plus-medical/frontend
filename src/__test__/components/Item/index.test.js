/* eslint-disable no-undef */
import React from 'react'
import { create } from 'react-test-renderer'
import RouterMock from '../../../__mocks__/RoutesMocks'
import Item from '../../../components/Item/index'

describe('<Item />', () => {
  test('Review Snapshot in Item component', () => {
    const photo = create(
      <RouterMock>
        <Item />
      </RouterMock>
    )
    expect(photo.toJSON()).toMatchSnapshot()
  })
})
