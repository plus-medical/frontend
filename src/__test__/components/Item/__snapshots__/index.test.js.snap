// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Item /> Review Snapshot in Item component 1`] = `
<a
  href="/user/undefined"
  onClick={[Function]}
>
  <li
    className="item"
  >
    <div
      className="item__figure"
    >
      <img
        alt="photo"
        className="item__image"
        src="https://pbs.twimg.com/profile_images/1055299470148227072/xeEopDm__400x400.jpg"
      />
      <svg
        className="item__icon"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M400 32H48C21.49 32 0 53.49 0 80v352c0 26.51 21.49 48 48 48h352c26.51 0 48-21.49 48-48V80c0-26.51-21.49-48-48-48zm0 400H48V80h352v352zm-35.864-241.724L191.547 361.48c-4.705 4.667-12.303 4.637-16.97-.068l-90.781-91.516c-4.667-4.705-4.637-12.303.069-16.971l22.719-22.536c4.705-4.667 12.303-4.637 16.97.069l59.792 60.277 141.352-140.216c4.705-4.667 12.303-4.637 16.97.068l22.536 22.718c4.667 4.706 4.637 12.304-.068 16.971z"
        />
      </svg>
    </div>
    <article
      className="item__content"
    >
      <p
        className="item__name"
      />
      <p
        className="item__dni"
      >
        undefined undefined
      </p>
      <p
        className="item__profile"
      />
    </article>
    <section
      className="item__indicator"
    />
  </li>
</a>
`;
